### ISSUE TITLE
No Mazebreaker in vault?

### ISSUE THREAD

**Issue by Posivated** [Darkosto/SevTech-Ages#1614]
Broke into secret vault and no Mazebreaker to be found?

**Comment by Bendder** [comment]
Did the TnT trap go off?  If it did you'll have to try again in a different maze.. :(

**Comment by Posivated** [comment]
No, I got all the loot from the 4 chests without setting off TNT

**Comment by Targren** [comment]
It's not a guaranteed reward, AFAIK. The RNG got you this time

**Comment by SummitSummit** [comment]
Brutal.  I got two in my first vault.  Do you want one?
